examples/gke/cluster.yaml

```diff
@@ -87,6 +87,7 @@ spec:
     name: <gcp_zone>
     racks:
       - name: <gcp_region>
+        scylla_config_map: "scylla-config"
```

Comment:
Kubernetes Objects use camelCase so I believe it'd be better to stick to the convention.
You can read more on the Kubernetes API Conventions here:
https://github.com/kubernetes/community/blob/2a656f46492fb074cdb36d8413721db52078654b/contributors/devel/sig-architecture/api-conventions.md

Reply:
Ok, fixing.